server/views/partials/_timeline.njk

```diff
@@ -19,6 +19,14 @@
         <p class="moj-timeline__date">
           <time datetime="{{event.datetime.timestamp}}">{{ event.datetime.date }}</time>
         </p>
+
+        {% if showDescription %}
```

Comment:
I'm getting a bit confused following the flag around - I wonder if the logic could be outsourced to the `util` function so there's one place that decides what's in the timeline? And then the view just conditionally renders `if event.description.text` and doesn't have to know about the flag? I guess the description is always going to be nullable so we have to account for that anyway?

Reply:
That's a lot neater 👍🏾